src/firebolt/common/util.py

```diff
@@ -37,15 +51,52 @@ def fix_url_schema(url: str) -> str:
     return url if url.startswith("http") else f"https://{url}"
 
 
-def async_to_sync(f: Callable) -> Callable:
+class AsyncJobThread:
+    def __init__(self) -> None:
```

Comment:
Please add a docstring as to why we have this class in the first place. I have a feeling I'll forget in a few month time what was this about and a docstring would be helpful.

Reply:
Yes, sure